core/runtime/src/main/java/io/quarkus/runtime/Quarkus.java

```diff
@@ -239,15 +239,15 @@ public static void manualInitialize() {
                 return;
             if (tmpState == MANUAL_FAILURE)
                 throw new RuntimeException("Quarkus manual bootstrap failed");
-            manualState = tmpState = MANUAL_BEGIN_INITIALIZATION;
```

Comment:
You drop it because we don't care about it past that line right?

Reply:
Yes, this extra write to the variable is never needed